other/upgrade_2-1_mysql.sql

```diff
@@ -1022,22 +1022,22 @@ if (!empty($select_columns))
 		while ($row = $smcFunc['db_fetch_assoc']($request))
 		{
 			if (!empty($row['aim']))
-				$inserts[] = array($row['id_member'], -1, 'cust_aolins', $row['aim']);
+				$inserts[] = array($row['id_member'], 0, 'cust_aolins', $row['aim']);
```

Comment:
i looked on the wrong col,
the -1 is in theme_id col...
here are -1 value not allowed -> 0 or even 1 could be right